conf/machine/imx8mm-ddr4-evk.conf

```diff
@@ -6,7 +6,7 @@
 
 require include/imx8mm-evk.inc
 
-KERNEL_DEVICETREE_BASENAME = "${MACHINE}"
+KERNEL_DEVICETREE_BASENAME = "imx8mm-ddr4-evk"
```

Comment:
Please adjust the commit lock to use the machine name as prefix and use the message that you sent and the pull request as the commit message for more detail